scripts/themes/index.js

```diff
@@ -21,16 +21,21 @@ const { value: themeMap } = renderSync({
 const themes = Object.keys(themeMap);
 
 // Generates the data for themes and tokens to template with Handlebars.
-const tokens = Object.keys(themeMap[themes[0]].value).map(token => ({
-  themes: [],
-  token,
-}));
+const tokens = Object.keys(themeMap[themes[0]].value)
+  // TODO - currently only handling color tokens.
+  .filter(token => themeMap[themes[0]].value[token].type === 'SassColor')
+  .map(token => ({
+    themes: [],
+    token,
+  }));
 
 // Loops through each of the supported themes, finding and formatting the correct color hex values and names.
 themes.forEach(theme => {
   const { value: tokenObject } = themeMap[theme];
 
   Object.keys(tokenObject)
+    // TODO - currently only handling color tokens.
+    .filter(tokenValue => tokenObject[tokenValue].type === 'SassColor')
     .map(tokenValue => tokenObject[tokenValue])
     .forEach(({ value }, index) => {
       let { hex: color = 'colors' } = value;
```

Comment:
- [ ] We can remove this default assignment because we're only outputting color values

Reply:
You don't want to keep it just in case?

Reply:
The purpose of the value was to replicate Carbon's themes (https://carbon-elements.netlify.com/themes/examples/preview/) - if we're filtering them out, I don't think we need this anymore